kong/plugins/rate-limiting/handler.lua

```diff
@@ -64,6 +68,30 @@ local function get_usage(conf, api_id, identifier, current_timestamp, limits)
   return usage, stop
 end
 
+local function get_period_end_in_secs(current_timestamp, limits)
+  local timetable
+  local periods = timestamp.get_timestamps(current_timestamp)
+
+  for period, period_date in pairs(periods) do
+    if limits[period] then
+      timetable = new_from_timestamp(floor(period_date / 1000))
+
+      -- Unfortunately second & minute don't map correctly
```

Comment:
I'd suggest changing this comment to "'second' and 'minute' are 'sec' and 'min' in timetable"